app/src/main/java/com/example/parkingsystem/mvp/view/ParkingView.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.parkingsystem.mvp.view
+
+import android.app.Activity
+import android.widget.Toast
+import com.example.parkingsystem.R
+import com.example.parkingsystem.mvp.contract.ParkingContract
+
+class ParkingView(activity: Activity) : ParkingContract.View, ActivityView(activity) {
+    override fun showParkingSizeToast(parkingSize: Int) {
+        Toast.makeText(
+            context,
+            context?.getString(R.string.txt_parking_size_toast,parkingSize),
+            Toast.LENGTH_LONG
+        ).show()
+    }
+
+}
```

Comment:
to avoid this error add an empty line at the end of the file

Reply:
Done 👍